oak-run-commons/src/main/java/org/apache/jackrabbit/oak/fixture/DataStoreUtils.java

```diff
@@ -146,26 +150,52 @@ public static void deleteBucket(String bucket, Map<String, ?> map, Date date) th
     }
 
     public static void deleteAzureContainer(Map<String, ?> config, String containerName) throws Exception {
+        if (config == null) {
+            return;
+        }
         if (Strings.isNullOrEmpty(containerName)) {
             return;
         }
-        String connectionString = (String) config.get(AzureConstants.AZURE_CONNECTION_STRING);
-        if (Strings.isNullOrEmpty(connectionString)) {
-            String accountName = (String) config.get(AzureConstants.AZURE_STORAGE_ACCOUNT_NAME);
-            String accountKey = (String) config.get(AzureConstants.AZURE_STORAGE_ACCOUNT_KEY);
-            if (Strings.isNullOrEmpty(accountName) ||
-                Strings.isNullOrEmpty(accountKey)) {
-                return;
-            }
-            connectionString = org.apache.jackrabbit.oak.blob.cloud.azure.blobstorage.Utils.getConnectionString(accountName, accountKey);
+        CloudBlobContainer container = getCloudBlobContainer(config, containerName);
+        if (container == null) {
+            log.info("container is not initialized");
```

Comment:
maybe change this to warn and a bit more descriptive.

Container not initialised while trying to delete it.

Reply:
This should be changed to warn.

Reply:
done